File: Python/outlier.py
def find(numbers):
    even = []
    odd = []

    for i in numbers:
        if i%2==0:
            even.append(i)
        else:
            odd.append(i)
    if len(even)>1:
        return odd[0]
    else:
        return even[0]

File: PHP/outlier.php
<?php 

    function find($integers){
        $odd = [];
        $even = [];
        foreach($integers as $integer){
            if($integer % 2 === 0){
                $even[] = $integer;
            }else{
                $odd[] = $integer;
            }
        }

        if(count($even)>1){
            return $odd[0];
        }else{
            return $even[0];
        }
    }